.github/workflows/e2e.yml

```diff
@@ -0,0 +1,45 @@
+name: Run e2e tests
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
e2e tests are much needed. Should we run them as part of each PR? Just like we do on Nx. They wouldn't be just useful for migration scenarios, but also to make sure PRs with new recipes that get merged have all their tests & e2e tests passing.